README.md

```diff
@@ -11,31 +11,45 @@ mm-plugin-apps: `master` @831d203
 
 ### Zendesk and Mattermost Users (System Admin privileges required)
 
-1. Clone this repo
-1. Create Zendesk Oauth Client for Mattermost (in Zendesk)
-    1. `Zendesk` > `Admin` > `API` > `OAuth Clients`
-    1. `Add OAuth Client`
-        1. `Client Name`: (Example `Mattermost Zendesk App`)
-        1. `Description`: `Connect your Zendesk account to Mattermost`
-        1. `Redirect URLs`: `https://<your-zendesk-app-host>/oauth/complete`
-            1. Ex. `http://localhost:4000` - Development
-            1. Ex. `https://mytest.ngrok.io` - Exposed for development
-        1. `Save`
-1. Install the app (In Mattermost)
-    1. `/apps install --url http://<your-zendesk-app-host>/manifest.json --app-secret thisisthesecret`  
-1. Configure Zendesk Client in Mattermost
-    1. `/zendesk configure` to open the configuration modal
-    1. Save values from Oauth Client form to the fields in the configuration modal
-        1. `Zendesk URL` - set to your zendesk account host
-        1. `Zendesk Client ID` - set to the `Unique identifier` field value
-        1. `Zendesk Client Secret` - set as the `Secret` field value
-            1. Ex. `https://<subdomain>.zendesk.com`
-        1. `ZD_NODE_HOST` - set to the path of your zendesk app host
-            1. Ex. `https://testing.ngrok.io`
-1. [Setup Zendesk Target](#Setup Zendesk Target)
-1. Start the node server
-    1. `make watch` - (to monitor typescript errors and watch changing files errors)
-    1. `make run` - (in a separate shell) start the node server
+##### 1. Create Zendesk Oauth Client for Mattermost (in Zendesk)
+
+1. `Zendesk` > `Admin` > `API` > `OAuth Clients`
+1. `Add OAuth Client`
+    1. `Client Name`: (Example `Mattermost Zendesk App`)
+    1. `Description`: `Connect your Zendesk account to Mattermost`
+    1. `Redirect URLs`: `https://<your-zendesk-app-host>/oauth2/complete`
+        1. Ex. `http://localhost:4000` - Development
+        1. Ex. `https://mytest.ngrok.io` - Exposed for development
```

Comment:
Is that correct? Does that path have to contain the Apps plugin id?

Reply:
@hanzei, with the latest Oauth2 changes, the redirect URL now points to the `mattermost_site_url`.  I've updated the README.